scripts/common.py

```diff
@@ -803,6 +805,10 @@ def managed_process(command_args, shell=False, timeout_secs=60, **kwargs):
         timeout_secs: int. The time allotted for the managed process and its
             descendants to terminate themselves. After the timeout, any
             remaining processes will be killed abruptly.
+        proc_name_to_kill: str|None. If given, the name of a string which
+            matches all additional processes that should be killed when the
+            given process is terminated. (This is needed for, e.g.,
+            elasticsearch.)
```

Comment:
I didn't made the changes in this file. I don't know why they are showing these.